### PR TITLE
Add "inRange" function solution

### PR DESCRIPTION
**Pull request by KuzyaCat**
Issue #31

### CODE REVIEW

File: classworks/alexander/01_git/result.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+https://github.com/KuzyaCat/git-learn/tree/master
```
Comment (dzinrai) [inline classworks/alexander/01_git/result.txt:1]: Unnecessary change

Comment (KuzyaCat) [reply classworks/alexander/01_git/result.txt]: fixed
